### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Arthur Jacobs

### DISCUSSION THREAD

**Discussion by Arthur2479** [heig-vd-dai-course/heig-vd-dai-course#335] [Show and tell]
https://github.com/Arthur2479/dai-java-intellij-and-maven

**Comment by ludelafo** [comment]
It seems I cannot access the repository. You can open it or if you want us to access it, you must add us to your project.

**Reply by Arthur2479** [reply]
You should have access now !

**Reply by ludelafo** [reply]
Perfect, thank you!

Everything seems to be OK, except the `pom.xml` file. You must only keep the `maven-shade-plugin`. The `maven-jar-plugin` can be removed to avoid any conflicts between the two packaging plugins. :)

**Reply by Arthur2479** [reply]
Noted tanks!
